setupcfg2nix/cli.py

```diff
@@ -1,6 +1,56 @@
-from __future__ import print_function
 from setuptools.config import read_configuration
 import argparse
+import unittest.mock
+import errno
+import os
+import os.path
+import tempfile
+import setuptools
+import sys
+from distutils.errors import DistutilsFileError
+
+requires_sets = [ "install_requires", "setup_requires", "tests_require" ]
+
+def make_declarative(package_dir, out_file):
+    cur_dir = os.getcwd()
+    cur_path = sys.path
+    try:
+        os.chdir(package_dir)
+        sys.path = sys.path[:]
+        sys.path.insert(0, '.')
+        with unittest.mock.patch.object(setuptools, 'setup') as mock_setup:
+            import setup
+
+            args, kwargs = mock_setup.call_args
+    finally:
+        os.chdir(cur_dir)
+        sys.path = cur_path
+
+    print("[metadata]", file=out_file)
```

Comment:
Is there a reason you're printing to the file object instead of simply doing `out_file.write(...)`?